docs/layers.md

```diff
@@ -1354,55 +1451,688 @@ Depending on OpenStreetMap tagging, the following properties may be present for
 
 Transit lines may have their colours mapped onto one of these CSS colours. The intention is that designers can take this limited palette set and remap them onto a set which is more appropriate and in keeping with the other colours in the design. Inspired by [py-cooperhewitt-swatchbook](https://github.com/cooperhewitt/py-cooperhewitt-swatchbook).
 
-* aqua
-* aquamarine
-* black
-* blue
-* brown
-* crimson
-* darkgrey
-* darkorchid
-* darkred
-* darkseagreen
-* dodgerblue
-* fuchsia
-* gainsboro
-* gold
-* goldenrod
-* green
-* grey
-* hotpink
-* indigo
-* khaki
-* lightblue
-* lightcoral
-* lightgreen
-* lightteal
-* lime
-* limegreen
-* mediumpurple
-* mediumseagreen
-* mediumturquoise
-* navy
-* olivedrab
-* orange
-* orangered
-* peru
-* pink
-* plum
-* purple
-* red
-* royalblue
-* sandybrown
-* silver
-* steelblue
-* tan
-* teal
-* tomato
-* violet
-* white
-* yellow
-* yellowgreen
+* `aqua`
+* `aquamarine`
+* `black`
+* `blue`
+* `brown`
+* `crimson`
+* `darkgrey`
+* `darkorchid`
+* `darkred`
+* `darkseagreen`
+* `dodgerblue`
+* `fuchsia`
+* `gainsboro`
+* `gold`
+* `goldenrod`
+* `green`
+* `grey`
+* `hotpink`
+* `indigo`
+* `khaki`
+* `lightblue`
+* `lightcoral`
+* `lightgreen`
+* `lightteal`
+* `lime`
+* `limegreen`
+* `mediumpurple`
+* `mediumseagreen`
+* `mediumturquoise`
+* `navy`
+* `olivedrab`
+* `orange`
+* `orangered`
+* `peru`
+* `pink`
+* `plum`
+* `purple`
+* `red`
+* `royalblue`
+* `sandybrown`
+* `silver`
+* `steelblue`
+* `tan`
+* `teal`
+* `tomato`
+* `violet`
+* `white`
+* `yellow`
+* `yellowgreen`
+
+#### Network values
+
+Any road with `shield_text` will include a `network` property with a value like `AA:bcdef` where `AA` is a 2-character country code, followed by a `:` separator, and `bcdef` category value which either indicates the "region" (state or province) or "level" of the road network. There are exceptions to this for trans-national networks like `e-road`. In some cases, like the United States and Canada an additional _modifier_ to appended to indicate `Truck` and other alternate routes, or further network disambiguation.
+
+Some countries without network tags but with ref values with `;` and `/` and other separators, including: Switzerland, Greece, India, Italy, Japan, Russia, Turkey, Vietnam, and South Africa.
+
+When we don't see network we backfill based on common road operators values. Network values always replace plural `??:roads` with singular `??:road`.
+
+When a network value can't be determined from the upstream data source we calculate where the road is located and provide the relevant 2-char country code as the network value. See table below for mapping of country codes to country names.
+
+Network value include:
```

Comment:
This listing is so detailed that in many places it duplicates the code. I worry that this will make it harder to understand for anyone reading it and harder for us to maintain as the code changes.

Reply:
(from IRL) I'll simplify this listing so it's mostly just the network values... and add new documentation page listing out these details – which are important because these aren't straight pass thrus of OSM values and it's useful to detail that, just not in the generic spec.

Reply:
Fixed in https://github.com/tilezen/vector-datasource/pull/1593/commits/61d508d8b6600f47ff834c89194324cf4e918007.